docs/installation.md

```diff
@@ -0,0 +1,45 @@
+# Installation
+
+To install this application you should search for "Elastic Agent" in the GCP Marketplace and install it through the GCP Console.
+
+From the product listing you can choose the version of the application to install. We maintain `7.17.x` and `8.x` stack versions.
+
+By clicking "Configure" a guided process starts to collect some deploymenta details.
+
+You will need to choose:
+- the GKE cluster where to deploy the application: you can select an existing cluster or create a new one;
+- the namespace within the GKE cluster; selecting `default` will deploy the Application to the `default` namespace and the Elastic Agent in the (`kube-system`);
+- the application instance name (this is the name of the deployed application in your GKE cluster);
+- the Service Account of the application; **do not change this field**, a dedicated Service Account with required permissions will be created;
+- Fleet Server URL to connect to; if left empty Kibana Host, Kibana Fleet Username and Kibana Fleet Password are needed;
+- Fleet enrollment token; is an Elasticsearch API key to enroll one or more Elastic Agents in Fleet. See: https://www.elastic.co/guide/en/fleet/current/fleet-enrollment-tokens.html. If left empty Kibana Host, Kibana Fleet Username and Kibana Fleet Password are needed;
+- Kibana Username; user name to connect to Kibana;
```

Comment:
Is there specific permissions that the user needs to have? Can you please elaborate as appropriate.